docs/src/pages/playground.module.css

```
@@ -2,7 +2,7 @@
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  height: 75vh;
+  min-height: 82vh;
   margin: 20px;
```

```suggestion
  margin: 20px;
  flex: 1;
```
to remove that empty space under